examples/nD_points_with_properties.py

```diff
@@ -19,26 +19,26 @@
 for z in range(blobs.shape[0]):
     points += [[z, 25, 25], [z, 25, 75], [z, 75, 25], [z, 75, 75]]
 
-# create the property for setting the face and edge color.
-face_property = np.array(
+# create the features for setting the face and edge color.
```

Comment:
Rename this file to `nD_points_with_features.py`?

Reply:
oh yeah, missed that, :+1: